tests/E2E/ErrorTest.php

```diff
@@ -40,20 +40,23 @@ public function testAccessTokenInvalid()
      */
     public function testInvalidQuery()
     {
-        $client = new Client('b4c0n73n7fu1', 'cfexampleapi');
+        $client = $this->getClient('cfexampleapi');
+
         $query = (new Query())
             ->where('name', 0);
 
         $client->getEntries($query);
     }
 
     /**
+     * @requires Contentful
```

Comment:
Fair enough. Just trying to make it clear we're not talking about normal HTTP proxies. (It'd be fun to rewrite the coverage tool being an actual proxy!)

`@requires API no-coverage-proxy`?